test/unit/calculators/benefit_cap_calculator_configuration_test.rb

```diff
@@ -127,7 +127,7 @@ class BenefitCapCalculatorConfigurationTest < ActiveSupport::TestCase
           end
         end
         context "merge questions" do
-          should "get all benefit questions from multiple configuration sets" do
+          should "get all unique benefit questions from multiple configuration sets" do
```

Comment:
Ideally this change would've been incorporated into 70ba9b321c03bc4da0faf0cac9d53bea80c7675b where the test was originally added.